samples/web/content/testrtc/js/main.js

```diff
@@ -418,23 +424,21 @@ function setTimeoutWithProgressBar(timeoutCallback, timeoutMs) {
 }
 
 // Parse URL parameters and configure test filters.
-{
```

Comment:
That is indeed true! My mistake.

Can you have a proper parserUrlParameters function? Instead of var parseUrlParameters = function ?